flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": ["x86_64", "aarch64"],
```

Comment:
```suggestion
```

there is no other arch

Reply:
I simply left the file as in the .NET 6 SDK repository and changed the minimum necessary for the new major version. But I also agree with removing it since, as you said, there are no other architectures to distribute.